src/signals/incident/components/form/MapSelectors/Clock/ClockSelectRenderer/ClockSelectRenderer.tsx

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2020 - 2021 Gemeente Amsterdam
+import type { FunctionComponent } from 'react'
+import FormField from '../../../FormField'
+import type { FormFieldProps } from '../../../FormField/FormField'
+import AssetSelect from '../../Asset'
+import Layer from '../ClockLayer'
+import type { Meta } from '../../Asset/types'
+
+interface ClockSelectRendererProps extends FormFieldProps {
+  meta: Meta
+  handler: any
+  parent: any
+  validatorsOrOpts: any
+}
```

Comment:
Can't we create an interface that can be used for all `SelectRenderer` components?

Reply:
good point